### PR TITLE
fix: marker image,loading animation,testimonials, meta POL

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/components/molecules/m-mapbox.vue [hunk 1]
```diff
@@ -29,12 +30,16 @@ export default {
     for (let i = 0; i < this.data.length; i++) {
       // geojson.features.forEach(function (marker) {
       //   // create a HTML element for each feature
-      //   var el = document.createElement("div");
-      //   el.className = "marker";
+      var el = document.createElement("div");
```
Comment (arrogant-ant) [inline src/components/molecules/m-mapbox.vue:33]: please use a better and descriptive name than `el` may be `markerElement`

File: src/components/molecules/m-mapbox.vue [hunk 2]
```diff
@@ -29,12 +30,16 @@ export default {
...
+      el.style.backgroundImage = "url(" + this.img + ")";
+      el.style.width = "60px";
+      el.style.height = "60px";
+      el.style.backgroundSize = "100%";
 
       // make a marker for each feature and add it to the map
-      var marker = new mapboxgl.Marker({
-        color: "#ffdd57",
+      var marker = new mapboxgl.Marker(el, {
+        // color: "#ffdd57"
```
Comment (arrogant-ant) [inline src/components/molecules/m-mapbox.vue:42]: remove this comment

File: src/components/molecules/m-mapbox.vue (hunk 1 above)
Comment (alias01) [reply src/components/molecules/m-mapbox.vue:33]: done

File: src/components/molecules/m-mapbox.vue (hunk 2 above)
Comment (alias01) [reply src/components/molecules/m-mapbox.vue:42]: done

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**